cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/CentralOAuthServiceImpl.java

```diff
@@ -286,32 +291,33 @@ public Boolean revokeToken(final Token token) {
         return ticketRegistry.deleteTicket(token.getTicket().getId());
     }
 
-    @Override
-    public Boolean revokeClientTokens(final String clientId, final String clientSecret) {
-        final OAuthRegisteredService service = getRegisteredService(clientId);
-        if (service == null) {
-            LOGGER.error("OAuth Registered Service could not be found for clientId : {}", clientId);
-            return Boolean.FALSE;
-        }
-        if (!service.getClientSecret().equals(clientSecret)) {
-            LOGGER.error("Invalid client secret");
-            return Boolean.FALSE;
-        }
-
-        final Collection<RefreshToken> refreshTokens = tokenRegistry.getClientTokens(clientId, RefreshToken.class);
-        for (final RefreshToken token : refreshTokens) {
-            LOGGER.debug("Revoking refresh token : {}", token.getId());
-            ticketRegistry.deleteTicket(token.getTicket().getId());
-        }
-
-        final Collection<AccessToken> accessTokens = tokenRegistry.getClientTokens(clientId, AccessToken.class);
-        for (final AccessToken token : accessTokens) {
-            LOGGER.debug("Revoking access token : {}", token.getId());
-            ticketRegistry.deleteTicket(token.getTicket().getId());
-        }
-
-        return Boolean.TRUE;
-    }
+//    TODO: Deprecated. Please remove.
+//    @Override
+//    public Boolean revokeClientTokens(final String clientId, final String clientSecret) {
```

Comment:
Similarly to the interface, the implementation of `revokeClientTokens()` is deprecated.